com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/dao/ConfigurationEntryDtoDao.java

```diff
@@ -108,6 +117,15 @@ public void remove(long id) throws NotFoundException {
         });
     }
 
+    @SuppressWarnings("unchecked")
+    public List<ConfigurationEntryDto> find(String guid) {
+        return new Executor<List<ConfigurationEntryDto>>(createEntityManager()).execute((manager) -> {
+
+            return manager.createNamedQuery(NamedQueries.FIND_ALL_ENTRIES_GUID).setParameter(ConfigurationEntryDto.FieldNames.SPACE_ID,
```

Comment:
Rename Named.Queries.FIND_ALL_ENTRIES_GUID to NamedQueries.FIND_ALL_ENTRIES_BY_SPACE_GUID.

Reply:
Or even better: Rename all 'guid' to 'id', since the field is called spaceId and not spaceGuid. Using two names for the same thing is not a good pattern.